frontend-react/src/pages/error/legacy-content/NoPage.tsx

```diff
@@ -0,0 +1,68 @@
+import { Helmet } from "react-helmet";
+import DOMPurify from "dompurify";
+import React from "react";
+
+import site from "../../../content/site.json";
+
+export const NoPage = () => {
```

Comment:
NoPage is _explicitly_ for when a page is not found (similar to the one-off use of UnsupportedBrowser). The two components that don't fit in `Generic` don't fit because of their custom styling and/or action buttons. The general idea is that content to hydrate `GenericError` is dynamically chosen based on a type of error OR maybe integrated into the error object shape/creation. There'll be a configuration with the content for a 404. I think communicating a page not found as a 404 or "not found" gets confusing for _us_, but also when someone tries to report a 404 that isn't really a 404, but might just be them typing their url in wrong. 🤷🏼

Reply:
Ahhh, I get it: I went ahead to name them `ErrorNoPage` and `ErrorUnsupportedBrowser` just to clarify they're bespoke (and legacy) error pages